static/src/js/components/DraftList/DraftList.jsx

```diff
@@ -53,6 +53,38 @@ const DraftList = ({ draftType }) => {
     })
   }
 
+  // Building a Table using Data in items
+  const renderRows = (items.map((item) => {
+    const { conceptId, revisionDate, previewMetadata: { name, longName } } = item
+    const draftLink = `/drafts/${`${paramDraftType}`}/${conceptId}`
+
+    return (
+      [
+        <Link key={`${conceptId}/col1`} to={draftLink}>
```

Comment:
I think probably the keys can be included in the Table.jsx as well, this way you are just passing it a 2D array of data.